src/main/java/com/flat/odessa/container/LinkAccumulator.java

```diff
@@ -0,0 +1,34 @@
+package com.flat.odessa.container;
+
+import com.flat.odessa.callback.Callback;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LinkAccumulator implements Accumulator<Collection<String>> {
```

Comment:
@greySlon I'd suggest to skip accumulate phase for now. The only client code needs is to be able to iterate over available links, which `Links` contract covers with its `iterator` method, right?